### ISSUE TITLE
MacOS Graphic Support

### ISSUE THREAD

**Issue by zzhou292** [ProjectPhysX/FluidX3D#88]
Hello,

First of all - this is an amazing software!

May I ask if there is a graphic support for M1 Mac yet? I couldn't get interactive graphics working on Mac....

Is there a possibility to export image files without interacting with it?

Thank you again!

JZ

**Comment by zzhou292** [comment]
I solved the problem myself.

For anyone else using Mac M1 or M2, see example at line 358.

Thank you for making the software!!!

**Comment by thilojaeggi** [comment]
Any clue? Line 358 doesn't help if we don't know the file...

**Comment by gitcnd** [comment]
If anyone wants to collaborate on what looks like relatively minor effort to get this working on Mac, let me know.  I've only got an intel Mac, and it would be nice to make it work on all of them.  I've already modified the windows version allowing it to run inside a window (instead or as well as full screen), and I had a quick try for my intel mac... so I *think* I've got a reasonable grasp of what's needed to make this work.
